docs/reference/error-codes.mdx

```diff
@@ -286,7 +286,7 @@ Set `validate: {creationStack: false}` in the options to disable the check.
 > Added in `6.10.0`.
 
 In express-rate-limit version 6 and older, the rate limiter would be disabled
-when setting `limit` (now `max`) to `0` in the options. Starting with version
+when setting `limit` to `0` in the options. Starting with version
```

Comment:
Oh 🤦 

Those two were supposed to be flipped. It's talking about older versions in that sentence, so it should read "max (now limit)"

```suggestion
when setting `max` (now `limit`) to `0` in the options. Starting with version
```